src/main/java/examples/VertxKafkaClientExamples.java

```diff
@@ -43,27 +43,116 @@
    * Example about Kafka consumer and producer creation
    * @param vertx
    */
-  public void example1(Vertx vertx) {
+  public void createConsumerJava(Vertx vertx) {
```

Comment:
Sorry ... other than keys even the values if possible (i.e. StringSerializer.class instead of the string)